constance/apps.py

```diff
@@ -6,6 +6,3 @@ class ConstanceConfig(AppConfig):
     name = 'constance'
     verbose_name = _('Constance')
     default_auto_field = 'django.db.models.AutoField'
-
-    def ready(self):
-        from . import checks
```

Comment:
this should be returned to enable checks
```
    def ready(self):
        from . import checks  # noqa: F401
```

Reply:
It shouldn't. It's not required for checks. Discussed in https://github.com/jazzband/django-constance/pull/562#discussion_r1664732150

Reply:
I resolve this to return to original comment